brainbeats-backend/Controllers/SampleController.cs

```diff
@@ -1,22 +1,29 @@
-﻿using System;
+﻿using System;
 using System.Collections.Generic;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
 using Newtonsoft.Json.Linq;
-using static brainbeats_backend.QueryStrings;
-using static brainbeats_backend.Utility;
-
-namespace brainbeats_backend.Controllers {
-  [Route("api/[controller]")]
-  [ApiController]
-  public class SampleController : ControllerBase {
-    [HttpPost]
-    [Route("create_sample")]
-    public async Task<IActionResult> CreateSample(dynamic req) {
-      JObject body = DeserializeRequest(req);
-
-      string queryString;
-
+using static brainbeats_backend.QueryStrings;
+using static brainbeats_backend.Utility;
+
+namespace brainbeats_backend.Controllers {
+  [Route("api/[controller]")]
+  [ApiController]
+  public class SampleController : ControllerBase {
+    [HttpPost]
+    [Route("create_sample")]
+    public async Task<IActionResult> CreateSample(dynamic req) {
+      try {
+        HttpContext.Request.Headers.TryGetValue("Authorization", out StringValues authorizationToken);
+        AuthConnection.Instance.ValidateToken(authorizationToken);
+      } catch {
+        return BadRequest("Unauthenticated");

```

Comment:
What are we catching in all these try/catches? Can there be other errors besides authentication? Shouldnt we be logging these?

Reply:
Updated to return specific error